com.unity.render-pipelines.universal/Runtime/Passes/ColorGradingLutPass.cs

```diff
@@ -183,8 +183,10 @@ public override void Execute(ScriptableRenderContext context, ref RenderingData
 
                 renderingData.cameraData.xr.StopSinglePass(cmd);
 
+                cmd.SetGlobalTexture(ShaderPropertyId.sourceTex, m_InternalLut);
+                CoreUtils.SetRenderTarget(cmd, m_InternalLut, RenderBufferLoadAction.DontCare, RenderBufferStoreAction.Store, ClearFlag.None, Color.clear);
                 // Render the lut
-                cmd.Blit(null, m_InternalLut.nameID, material);
+                cmd.Blit(null, BuiltinRenderTextureType.CurrentActive, material, 0);
```

Comment:
I'm confused why we still rely on BuiltinRenderTextureType after RTHandle. Why is that?